src/components/TimeBasedLineChart.js

```diff
@@ -272,9 +272,17 @@ const TimeBasedLineChart = React.createClass({
   },
 
   _yRangeContainsZero () {
-    const minMaxValues = ChartUtils.getDataMinMaxValues(this.props.data, 'y');
+    const max = d3.max(this.props.data, d => {
+      return d.y;
+    });
+
+    const min = d3.min(this.props.data, d => {
+      return d.y;
+    });
```

Comment:
What do you think about using the new shorthand style implicit returns to make these (and the ones in the other files) a tiny bit easier to read?
```javascript
const max = d3.max(this.props.data, d => d.y);
const min = d3.min(this.props.data, d => d.y);
```